packages/amplify-graphql-api-construct/tsconfig.json

```diff
@@ -1,11 +1,34 @@
 {
-  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "lib",
     "rootDir": "src",
+    "declarationMap": false,
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "alwaysStrict": true,
+    "declaration": true,
+    "experimentalDecorators": true,
+    "incremental": true,
+    "lib": ["es2020"],
+    "module": "CommonJS",
+    "noEmitOnError": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
     "strict": true,
-    "allowJs": false,
-    "removeComments": false
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
+    "stripInternal": false,
+    "target": "ES2020",
+    "composite": false,
+    "tsBuildInfoFile": "lib/tsconfig.tsbuildinfo"
   },
-  "exclude": ["coverage", "lib", "src/__tests__"]
+  "include": ["src/**/*.ts"],
+  "exclude": ["node_modules", "/Users/alharris/workplace/amplify-category-api/packages/amplify-graphql-api-construct/lib/.types-compat"],
```

Comment:
remove local workspace reference. Also, any reason to not exclude the `__tests__`?

Reply:
Yeah, similar to the response to Aaron's comment above, but this file is now auto-generated. I forgot to delete it in the first pass.